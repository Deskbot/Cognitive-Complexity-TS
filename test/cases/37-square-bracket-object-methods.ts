// would be too hard to detect recursion here
var something;
const o = {
    [something]() {
        if (true) {

        }
    }
}