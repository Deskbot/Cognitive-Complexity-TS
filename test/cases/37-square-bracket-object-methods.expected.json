{
    "37-square-bracket-object-methods.ts": {
        "score": 1,
        "inner": [
            {
                "name": "[something]",
                "score": 1,
                "line": 4,
                "column": 5
            }
        ]
    }
}
